octokit/meta.go

```diff
@@ -0,0 +1,41 @@
+package octokit
+
+import (
+	"net/url"
+
+	"github.com/jingweno/go-sawyer/hypermedia"
+)
+
+// https://developer.github.com/v3/meta/
+var (
+	MetaURL = Hyperlink("/meta")
+)
+
+// https://developer.github.com/v3/meta/
+func (c *Client) Meta(url *url.URL) *MetaService {
+	return &MetaService{client: c, URL: url}
+}
+
+// A service to return information about GitHub.com, the service.
+type MetaService struct {
+	client *Client
+	URL    *url.URL
+}
+
+// Get the user search results based on MetaService#URL
+//
+// https://developer.github.com/v3/meta/#meta
+func (m *MetaService) One() (meta Meta, result *Result) {
```

Comment:
Thanks for checking. `One` isn't a great fit here, is it? Wondering if we should forgo the service and just have `client.Meta` return an `ApiInfo` struct? Thoughts?